api/src/test/java/io/grpc/InternalTimeUtilsTest.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2024 The gRPC Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.grpc;
+
+import static io.grpc.InternalTimeUtils.convert;
+import static org.junit.Assert.assertEquals;
+
+import java.time.Duration;
+import org.junit.Test;
+
+public class InternalTimeUtilsTest {
```

Comment:
This is not how you write unit tests, it does not test InternalTimeUtils in anwyway. You need to call convert(Duration) on InternalTimeUtils and assert for the expected responses.

Reply:
How is this test testing InternalTimeUtils at all, when you haven't used that class?

Reply:
my bad, by oversight I didn't notice the static import for convert is from InternalTimeUtils class. I was only referring to InternalTimeUtilsTest and it is actually fine